Lib/email/_header_value_parser.py

```diff
@@ -2171,6 +2177,32 @@ def parse_message_id(value):
 
     return message_id
 
+def parse_message_ids(value):
+    """in-reply-to     =   "In-Reply-To:" 1*msg-id CRLF
+       references      =   "References:" 1*msg-id CRLF
+    """
+    message_ids = MessageIDList()
```

Comment:
To follow my coding pattern, this should be `message_id_list` rather than `message_ids`.  Not a big deal but it is nice to be consistent.